python/cudf/io/csv.py

```diff
@@ -103,6 +104,13 @@ def read_csv(filepath_or_buffer, lineterminator='\n',
         size to zero to read all data after the offset location. Reads the row
         that starts before or at the end of the range, even if it ends after
         the end of the range.
+    na_values : list, default None
```

Comment:
I missed that this parameter can be a dict. 
This would provide measurable perf benefits only if keep_default_na is false, otherwise every field has to be at least checked against the defaults.

>Do we intend to support that or no? If so, we could open up a separate Issue for it, with lower priority depending on how valuable it is.
@mjsamoht any idea about the priority? Adding this would complicate the code, but should not affect performance negatively, if node right.

Reply:
I think we can defer this to later, until a customer specifically asks for this.